HomeWork/Account.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace HomeWork
+{
+    public class Account
+    {
+        long _number = 8834_4686_3468_7843;        
+
+        decimal _balance = 0.10m;
+
+        TypeOfBankAccount _typeAccount = TypeOfBankAccount.DEBIT;
```

Comment:
```suggestion
        private long _number = 8834_4686_3468_7843;        

        private decimal _balance = 0.10m;

        private TypeOfBankAccount _typeAccount = TypeOfBankAccount.DEBIT;
```

На забывайте модификаторы доступа!